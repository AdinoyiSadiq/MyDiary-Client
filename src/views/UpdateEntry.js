import React from 'react';
import Main from '../components/layout/Main';

const UpdateEntry = (props) => (
  <div>
    <Main {...props}>
      This is the UpdateEntry view
    </Main>
  </div>
);

export default UpdateEntry;