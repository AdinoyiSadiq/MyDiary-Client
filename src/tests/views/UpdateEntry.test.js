import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import UpdateEntry from '../../views/UpdateEntry';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<UpdateEntry />);
});


describe('UpdateEntry UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});