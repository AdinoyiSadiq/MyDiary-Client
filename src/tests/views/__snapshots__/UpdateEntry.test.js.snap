// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpdateEntry UI render features view should render as expected 1`] = `
<div
  className="addEntryContent"
>
  <Main>
    <Connect(AddEntryForm)
      type="update"
    />
  </Main>
</div>
`;
